js/sales.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var firstAndPike = {
+  name: '1st and Pike',
+  minHourlyCustomer: 23,
+  maxHourlyCustomer: 65,
+  hourlyCookiesPerCustomer: 6.3,
+  cookiesPurchasedEachHour: [],
+  generateCustomers: function (min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+};
+
+var seaTacAirport = {
+  name: 'SeaTac Airport',
+  minHourlyCustomer: 3,
+  maxHourlyCustomer: 24,
+  hourlyCookiesPerCustomer: 1.2,
+  cookiesPurchasedEachHour: [],
+  generateCustomers: function (min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+};
+
+var seattleCenter = {
+  name: 'Seattle Center',
+  minHourlyCustomer: 11,
+  maxHourlyCustomer: 38,
+  hourlyCookiesPerCustomer: 3.7,
+  cookiesPurchasedEachHour: [],
+  generateCustomers: function (min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+};
+
+var capitolHill = {
+  name: 'Capitol Hill',
+  minHourlyCustomer: 20,
+  maxHourlyCustomer: 38,
+  hourlyCookiesPerCustomer: 2.3,
+  cookiesPurchasedEachHour: [],
+  generateCustomers: function (min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+};
+
+var alki = {
+  name: 'Alki',
+  minHourlyCustomer: 2,
+  maxHourlyCustomer: 16,
+  hourlyCookiesPerCustomer: 4.6,
+  cookiesPurchasedEachHour: [],
+  generateCustomers: function (min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+};
+
+var generateList = function (store) {
+  // Generate cookies for each hour from 6 AM to 8 PM, 15 hours in all
+  for(var i = 0; i < 15; i++) {
+    store.cookiesPurchasedEachHour[i] = store.generateCustomers(store.minHourlyCustomer, store.maxHourlyCustomer);
```

Comment:
Right now, you are pushing the average number of customers per hour into your array and displaying those numbers. You should be displaying the average number of cookies per hour. You need to take the number you are calculating and multiply it by the average number of cookies per customer!